src/components/designer/custom/Link.tsx

```diff
@@ -88,6 +88,16 @@ const CustomLink: React.FC<ILinkDefaultProps> = ({
     };
   }, [link.properties, theme]);
 
+  const opacity = (() => {
```

Comment:
Create a lower-opacity line here if channel is private.

Reply:
Done here: 60b888bf568a58ebe5e4de517a76157e27883d96.

Reply:
Hey, sorry to nitpick, but what I meant was to return `opacity` in the existing `useMemo` call that's already present on line 50. Since this block is already converting `link.properties` values into component vars, it makes sense to determine the `opacity` in there as well.

Something like this is what I had in mind:
```js
  const { leftStop, rightStop, leftColor, rightColor, opacity } = useMemo(() => {
    ...
    // the opacity of the link's path defaults to 100%
    let opacity = 1;
    if (link.properties) {
      const {
        type,
        direction,
        toBalance,
        capacity,
        isPrivate,
      } = link.properties as LinkProperties;
      if (type === 'open-channel') {
        ...
      }
      // use a lighter opacity for private channels
      if (isPrivate) opacity = 0.5;
    }
    return {
      leftStop,
      rightStop,
      leftColor,
      rightColor,
      opacity,
    };
  }, [link.properties, theme]);
```